pkg/project/version_bundle_kvm.go

```diff
@@ -8,10 +8,10 @@ var versionBundleKVM = versionbundle.Bundle{
 	Changelogs: []versionbundle.Changelog{
 		{
 			Component:   "cluster-operator",
-			Description: "TODO",
+			Description: "Support xxs, xs, and small cluster profile detection.",
```

Comment:
From what I see, I think only `kvm` logic is different. Can we describe this difference in changelog?

Reply:
Updated changelog with extra details, PTAL